pandas/tseries/tests/test_timeseries.py

```diff
@@ -2362,7 +2362,7 @@ def test_map(self):
         f = lambda x: x.strftime('%Y%m%d')
         result = rng.map(f)
         exp = [f(x) for x in rng]
-        self.assert_numpy_array_equal(result, exp)
+        np.testing.assert_array_equal(result, exp)
```

Comment:
ahh, you are not comparing numpy arrays (they are actually a list and an array), use ``exp = np.asarray(....)`` and I think it will work, you can also try ``tm.assert_almost_equal(...)``